src/main/java/com/blackducksoftware/integration/fortify/batch/util/VulnerabilityUtil.java

```diff
@@ -83,36 +83,36 @@
             String hubProjectVersion, Date maxBomUpdatedDate, PropertyConstants propertyConstants) {
         List<Vulnerability> vulnerabilities = new ArrayList<>();
         vulnerabilityComponentViews.forEach(vulnerableComponentView -> {
-            String[] componentVersionLinkArr = vulnerableComponentView.getComponentVersionLink().split("/");
+            String[] componentVersionLinkArr = vulnerableComponentView.componentVersionLink.split("/");
```

Comment:
Not sure if you want this to be a separate issue, but before releasing the newer version of this we should update this code so that it doesnt parse urls or generate ui links. That is entirely too fragile.

The UI will redirect a decent amount of the API URLs to UI URLs given an html media type. An API link should be used in place of a UI link.

Reply:
I agreed Rob. In Fortify on Demand, we are passing the API url to determine the unique key. But for Fortify SSC, the fortify parser is expecting the project id and version id. If we modify it in SSC plugin, then we need to modify the corresponding changes in SSC parser as well. Since it is a minor release, please let us know whether can we leave it as is and in future, if we get a major enhancement request, can we modify them.

Reply:
I thought we agreed to use urls instead of IDs?

Reply:
That is for Fortify on Demand Rob. This is Fortify SSC.